### DISCUSSION TITLE
Azure support

### DISCUSSION THREAD

**Discussion by sanmai-NL** [unikraft/unikraft#601] [Q&A] [answered]
A few cloud providers are [mentioned in the README](https://github.com/unikraft/unikraft/blob/9bf6e63314a401204c02597834fb02f63a29aaf4/README.md#%EF%B8%8F-iaas-providers). 

What's there to say about Azure support?

My research group is bound to Azure due to enterprise architecture.
I would love to be able to experiment with Unikraft products on Azure.

**Comment by marcrittinghaus** [comment, marked as answer]
Hi @sanmai-NL,

we currently do not support Azure (and Hyper-V as underlying hypervisor), but are actively pursuing this. At the moment this is a pure community effort. Nevertheless, we are making constant progress to run Unikraft on Hyper-V. Integration with the Azure Cloud infrastructure services would come next.

In a call yesterday with the respective community member, we asked him to provide a draft PR soon, so other members of the community and core developers can have a look.

**Reply by marcrittinghaus** [reply]
May I ask what applications or use cases you envision for Unikraft on Azure?

**Comment by JonasBorchelt** [comment]
Same story over here. Would be really nice if there would be Azure support. Big Thanks for your all effort!

**Reply by marcrittinghaus** [reply]
Many thanks for the feedback @JonasBorchelt. That helps us a lot when setting priorities. Since the last post the community member working on this has published a very early draft PR for Hyper-V support. However, most device drivers like network and block devices are still work in progress. We now have a second member working on this to hopefully speed up things a bit.

We will let you know, when there is a first version to try out!

**Reply by JonasBorchelt** [reply]
Hey Marc (@marcrittinghaus),

thats amazing. Thank you for the this and the future update!

Have a good one!

**Comment by razvand** [comment]
Everyone, since we are already (viciously) using [the Unikraft Discord server](https://bit.ly/UnikraftDiscord) for discussions, meetings, brainstorming and jokes (mostly bad, we do what we can), we will close the Discussions tab. We will do this on Friday, March 31, 2023.

Please move any discussions or questions on [the Unikraft Discord server](https://bit.ly/UnikraftDiscord). See you all there! :handshake:
